tools/testdata/check_format/cpp_std.cc

```diff
@@ -0,0 +1,12 @@
+#include <memory>
+
+#include "absl/memory/memory.h"
+
+namespace Envoy {
+
+std::unique_ptr<int> bad = absl::make_unique<int>(0);	
```

Comment:
thanks for adding the test data. But you'll need to also add some lines to tools/check_format_test_helper.py to run on this file with expected outputs. Scan to the end of that file and you'll see what to do.

Reply:
Thanks! Added